src/components/movies/card/MovieCard.js

```diff
@@ -3,22 +3,22 @@ import PropTypes from "prop-types";
 import classes from "./MovieCard.module.css";
 import Actions from "./actions/Actions";
 
-function MovieCard({movie, deleteMovie, editMovie}) {
+function MovieCard({movie, deleteMovie, editMovie, selectDescription}) {
```

Comment:
```suggestion
function MovieCard({movie, deleteMovie, editMovie, showDescription}) {
```
Такое название лучше подходит

Reply:
Renamed